before_install_linux.sh

```diff
@@ -15,35 +15,50 @@ if [ ! -d /usr/include/freetype ]; then
   sudo ln -sf /usr/include/freetype2 /usr/include/freetype
 fi
 
+dir_options=""
```

Comment:
Since we end up needing the build path in so many places, maybe it would make sense to assign it to a local:

```bash
project_dir=`pwd`
build_dir=${project_dir}/build-ImageMagick/ImageMagick-${IMAGEMAGICK_VERSION}
if [ -v CONFIGURE_OPTIONS ]; then
  build_dir=${build_dir}-${CONFIGURE_OPTIONS}
fi
```